tensorflow/lite/kernels/BUILD

```diff
@@ -637,6 +637,7 @@ BUILTIN_KERNEL_SRCS = [
     "strided_slice.cc",
     "sub.cc",
     "svdf.cc",
+    "table.cc",
```

Comment:
Is it possible to remove the new custom op from the default op registration? Instead, we can keep it as a part of custom_ops target under the tensorflow/lite/BUILD file.

Most of common TFLite users do not requires a new table custom op for now and we would like to keep the TFLite library as compact as possible.